pyproject.toml

```diff
@@ -71,17 +71,12 @@ vision = [
     "monai>=1.3.2",
 ]
 all = [
-    "h5py>=3.10.0",
-    "nibabel>=5.2.0",
-    "opencv-python-headless>=4.9.0.80",
-    "timm @ git+https://github.com/huggingface/pytorch-image-models.git@main",
-    "torchvision>=0.17.0",
-    "openslide-python>=1.3.1",
-    "gdown>=5.2.0",
-    "scikit-image>=0.24.0",
-    "imagesize>=1.4.1",
-    "scipy>=1.14.0",
-    "monai>=1.3.2",
+    "kaiko-eva[vision]",
+    "kaiko-eva[lint]",
+    "kaiko-eva[typecheck]",
+    "kaiko-eva[test]",
+    "kaiko-eva[docs]",
+    "kaiko-eva[dev]",
```

Comment:
this actually is not entirely correct, as it points to the pip version of the lib, and not the current local one, so the dependencies are not the same

issue: https://github.com/kaiko-ai/eva/issues/267
comment: https://github.com/pdm-project/pdm/issues/775#issuecomment-986212817
feature: https://github.com/pdm-project/pdm/issues/2701

Reply:
Good point. Reverted this now to manually copying the requirements.